tests/functional/services/compute/v2/test_get_task_batch.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+import pytest
+
+import globus_sdk
+from globus_sdk._testing import load_response
+
+
+@pytest.mark.parametrize(
+    "task_id_style", ("string", "list", "set", "uuid", "uuid_list")
+)
```

Comment:
In place of strings and a conditional with an `else: raise` block, I recommend using transformation lambdas directly. Below, I've added IDs so that if errors ever occur, pytest will clearly show which transformer failed:

```suggestion
@pytest.mark.parametrize(
    "transform",
    (
        pytest.param(lambda x: x, id="string"),
        pytest.param(lambda x: [x], id="list"),
        pytest.param(lambda x: {x}, id="set"),
        pytest.param(lambda x: uuid.UUID(x), id="uuid"),
        pytest.param(lambda x: [uuid.UUID(x)], id="uuid_list"),
    ),
)
```

Then, in the test itself, the conditional just collapses to

```python
task_ids = transform(task_ids)
```

Reply:
I like it!